mod-pubsub-client/src/test/java/org/folio/util/pubsub/PubSubClientUtilsTest.java

```diff
@@ -81,4 +82,8 @@ public void shouldThrowExceptionWhenReadInvalidMessagingDescriptor() throws IOEx
     PubSubClientUtils.readMessagingDescriptor();
   }
 
+  @Test
+  public void shouldBuildCorrectModuleId() {
+    assertThat(PubSubClientUtils.getModuleId(), matchesPattern("mod-pubsub-[0-9]{1}.+"));
```

Comment:
```suggestion
    assertThat(PubSubClientUtils.getModuleId(), matchesPattern("mod-pubsub-[0-9].+"));
```

Or maybe something even more specific like `mod-pubsub-\d+\.\d+\.\d+.*`